src/lib/support/ThreadOperationalDataset.cpp

```diff
@@ -42,6 +42,8 @@ class ThreadTLV final
                                                    ///< not allowed in Thread Operational Dataset TLVs.
```

Comment:
not code under review, but while we're here:

```suggestion
                                                   ///< is not allowed in Thread Operational Dataset TLVs.
```